app/models/multi_record_csv.rb

```diff
@@ -33,9 +33,11 @@ def to_csv
           # Get a list of all granules
           records_for_format.each do |record|
             associated_granule_value = record.associated_granule_value
-            next unless !associated_granule_value.nil? &&
-                        associated_granule_value != 'Undefined' &&
-                        associated_granule_value != 'No Granule Review'
+            if associated_granule_value.empty?        ||
```

Comment:
this section looks much better to my eyes

Reply:
agree

Reply:
However, I'm unclear if the `associated_granule_value` can be nil. If you want to catch both nil and empty, you should use `.blank?`, but if it can only be one but not the other, then you can use the single appropriate check